lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
They represent various kinds of non-breaking spaces not supported in Java's or C++'s whitespace checking methods. In order to compare against them, we probably would have to cast `c` to one of the bigger char types, but that doesn't make much sense to me, because if `c` couldn't hold those values when it came in, that means it will never equal one of them... Unless I'm understanding this wrong.